regression/cbmc-sequentialization/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
+  set(gcc_only "-X gcc-only")
+else()
+  set(gcc_only "")
+endif()
+
+add_test_pl_tests(
+    "$<TARGET_FILE:cbmc> --validate-goto-model --validate-ssa-equation" -X smt-backend ${gcc_only}
+)
+
+add_test_pl_profile(
+    "cbmc-paths-lifo"
+    "$<TARGET_FILE:cbmc> --paths lifo ${gcc_only}"
```

Comment:
I believe `${gcc_only}` should be outside the quotes here, but I'm not sure.

@hannes-steffenhagen-diffblue ?